commissioning.tex

```diff
@@ -33,47 +33,46 @@ \subsection{Commissioning Plan and Schedule}
 See Section 6 of \citeds{SITCOMTN-005} for the baseline design of the SV surveys.
 All plans for commissioning observations are subject to change based on system readiness up until the moment of execution. 
 
-The System Optimization and SV survey phases are expected to last about 8 weeks each.
 The project schedule will continue to evolve as the remaining subcomponents are delivered. 
 Construction is expected to complete and LSST data taking start before the end of 2025.
 
-\subsection{Commissioning Milestones}
+\subsection{Key Commissioning Milestones}
 \label{ssec:commissioning-milestones}
 
-Commissioning work is being planned around three major milestones, \textit{ComCam First Photon}, \textit{LSSTCam First Photon} and \textit{System First Light}. 
+Commissioning work is being planned around three major milestones, \textit{LSSTComCam First Photon}, \textit{LSSTCam First Photon} and \textit{Rubin First Light}. 
 
-\textbf{ComCam First Photon}: The first image of the night sky produced by photons passing through the Rubin optical system and detected by the Commissioning Camera (ComCam). 
-This milestone was achieved on 24 October 2024. 
+\textbf{ComCam First Photon}: The first image of the night sky produced by photons passing through the Rubin optical system and detected by the Commissioning Camera (LSSTComCam). 
+This milestone was achieved on \ccfpdate. 
 
 \textbf {LSSTCam First Photon}: The first image of the night sky produced by photons passing through the Rubin optical system and detected by the LSST Science Camera (LSSTCam).
-Currently expected  for 15 April 2025.
+This milestone was achieved on \lcfpdate. 
 
-\textbf {System First Light}: Defined as the point at which we can routinely acquire science-grade imaging across the LSSTCam full focal plane and have a well understood technical path towards meeting the Construction Completeness criteria   \citeds{sitcomtn-061}.
-Currently expected for 4 July 2025. 
+\textbf {Rubin First Light}: Defined as the point at which we can routinely acquire science-grade imaging across the LSSTCam full focal plane and have a well understood technical path towards meeting the Construction Completeness criteria   described in \citeds{sitcomtn-061}.
```

Comment:
```suggestion
\textbf{Rubin First Light}: Defined as the point at which we can routinely acquire science-grade imaging across the LSSTCam full focal plane and have a well understood technical path towards meeting the Construction Completeness criteria described in \citeds{sitcomtn-061}.
```

(Removing the blank after the \textbf will yield better formatting in GitHub, though it's irrelevant to LaTeX itself.)